src/video/cameradevice.cpp

```diff
@@ -55,7 +55,10 @@ CameraDevice* CameraDevice::open(QString devName, AVDictionary** options)
     CameraDevice* dev = openDevices.value(devName);
     int aduration;
     if (dev)
-        goto out;
```

Comment:
I agree about not using goto in general, but IMHO this is the exact case where it's usefull.